gammapy/analysis/core.py

```diff
@@ -56,7 +44,7 @@ class Analysis:
 
     def __init__(self, config=None):
         if isinstance(config, dict):
-            self._config = AnalysisConfig(config)
+            self._config = AnalysisConfig(**config)
         elif isinstance(config, AnalysisConfig):
             self._config = config
         else:
```

Comment:
Change:
```
ValueError: Dict or `AnalysiConfig` object required.
```
To:
```
TypeError: config must be dict or AnalysisConfig
```
lower-case "dict" and fix typo in "AnalysisConfig", and change to a `TypeError`.

Reply:
Done.